add example hydra config

misko opened the pull request:
Add a eSCN hydra config example

lbluque reviewed (commented):
Thanks @misko ! 

Can you move this to the top level configs directory and name it something like hydra_example.yml so its clear. Ideally if you can add a few comments just describing the options (similar to the ocp_example.yml) that would be great!

misko commented:
> Thanks @misko !
> 
> Can you move this to the top level configs directory and name it something like hydra_example.yml so its clear. Ideally if you can add a few comments just describing the options (similar to the ocp_example.yml) that would be great!

I moved it and changed to eSCN hydra instead of equiformer v2 , just to reduce the number of lines in the config and make it easier to see where hydra + heads are
